src/WebJobs.Script.WebHost/ContainerManagement/LinuxFunctionExecutionActivityPublisher.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.WebHost.Management;
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.ContainerManagement
+{
+    public class LinuxFunctionExecutionActivityPublisher : IHostedService, IDisposable, ILinuxFunctionExecutionActivityPublisher
+    {
+        private const int FlushIntervalMs = 5 * 1000; // 5 seconds
```

Comment:
5 seconds seems too frequent for this. I thought we had discussed something like 30s or 1m?

Reply:
Made it 20